.changeset/signal.md

```diff
@@ -0,0 +1,5 @@
+---
+"openapi-react-query": minor
```

Comment:
Yes this adds a feature, but for pre-1.0 we’re only releasing breaking changes in minor releases

```suggestion
"openapi-react-query": patch
```